torchao/prototype/safetensors/safetensors_utils.py

```diff
@@ -6,34 +6,42 @@
 import torch
 
 import torchao
-from torchao.quantization import Float8Tensor
+from torchao.quantization import Float8Tensor, Int4PreshuffledTensor, Int4Tensor
 from torchao.quantization.quantize_.common import KernelPreference
 from torchao.quantization.quantize_.workflows import QuantizeTensorToFloat8Kwargs
 
 ALLOWED_CLASSES = {
     "Float8Tensor": Float8Tensor,
+    "Int4Tensor": Int4Tensor,
+    "Int4PreshuffledTensor": Int4PreshuffledTensor,
```

Comment:
it's fine to not support this one actually, since we don't need to distribute this